app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -256,7 +250,17 @@ function FileInput(props, ref) {
           {hint}
         </span>
       )}
-      {statusProps && <span {...statusProps} />}
+      <StatusMessage
+        status={shownErrorMessage ? Status.ERROR : Status.SUCCESS}
+        className={
+          !shownErrorMessage &&
+          (successMessage === fileLoadingText || successMessage === fileLoadedText)
+            ? 'usa-sr-only'
+            : undefined
+        }
+      >
+        {shownErrorMessage || successMessage}
+      </StatusMessage>
```

Comment:
Looking at this... seems like we're basically repeating all these status ternaries inside the component. A few ideas:

1. separate success/error components?
2. one component with an `errorMessage` field and a `successMessage` field, and the component shows one or the other and styles itself appropriately?

Reply:
Yeah two separate instances make a lot more sense!

Reply:
Implemented this in 0d97868.